static/js/zamboni/devhub.js

```diff
@@ -990,7 +990,10 @@ function initCatFields(delegate) {
             $("input", $misc).prop("checked", checkedLength <= 0);
         };
         var checkOther = function() {
-            $("input", $main).prop("checked", false).prop("disabled", false);
+          // Save Android categories
+          var $androidCat = $main.find('input[name="form-1-categories"]');
+          $("input", $main).prop("checked", false).prop("disabled", false);
+          $($androidCat+":checked").prop("checked", true).prop("disabled", false);
```

Comment:
so should I try placing a <code>if</code> condition before <code>$($androidCat+":checked").prop("checked", true).prop("disabled", false);</code> to check whether 'form-1-categories' exists or not?